app/src/pages/trace/TracePage.tsx

```diff
@@ -572,6 +579,98 @@ function RetrieverSpanInfo(props: {
   );
 }
 
+function RerankingSpanInfo(props: {
+  span: Span;
+  spanAttributes: AttributeObject;
+}) {
+  const { spanAttributes } = props;
+  const rerankingAttributes = useMemo<AttributeObject | null>(() => {
+    const rerankingAttrs = spanAttributes[SemanticAttributePrefixes.reranking];
+    if (typeof rerankingAttrs === "object") {
+      return rerankingAttrs as AttributeObject;
+    }
+    return null;
+  }, [spanAttributes]);
+  const query = useMemo<string>(() => {
+    if (rerankingAttributes == null) {
+      return "";
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.query] ||
+      "") as string;
+  }, [rerankingAttributes]);
+  const input_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.input_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+  const output_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.output_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+
+  const numInputDocuments = input_documents.length;
+  const numOutputDocuments = output_documents.length;
+  return (
+    <Flex direction="column" gap="size-200">
+      <Card title="Query" {...defaultCardProps}>
+        <CodeBlock value={query} mimeType="text" />
+      </Card>
+      <Card
+        title={`Input Documents (${numInputDocuments})`}
+        {...defaultCardProps}
+        defaultOpen={false}
+      >
+        {
+          <ul
+            css={css`
+              padding: var(--ac-global-dimension-static-size-200);
+              display: flex;
+              flex-direction: column;
+              gap: var(--ac-global-dimension-static-size-200);
+            `}
+          >
+            {input_documents.map((document, idx) => {
+              return (
+                <li key={idx}>
+                  <DocumentItem document={document} />
+                </li>
+              );
+            })}
```

Comment:
curious if this is largely not useful if as you say as the retrieval ordering is not useful - or is the idea to show the `score` of the original retrieval alongside the final rank-ordered one? I think if it's just duplication of data it might make more sense to just nix this.

Reply:
Are you considering not including this in the semantic convention at all, or just not displaying it in the UI? If it's already included in the span then displaying it seems a natural thing to do, even though the ordering is not meaningful.

Reply:
I think it might largely be duplicate to the retriever but yeah we can leave it for now for transparency reasons. Bummer on the ordering. I think there will be asks about ordering but we can wait for feedback.

Reply:
I think it's just that ordering is not guaranteed in the general case, but should exist in most cases. On the other hand, if the `score` is missing for any reason (since it's not essential to the reranker), the ordering would be lost regardless, so it's also not reliable.